GenderPayGap.WebUI/Controllers/Admin/AdminDownloadsController.cs

```diff
@@ -96,8 +96,6 @@ public FileContentResult DownloadOrphanOrganisations()
                         Address = org.GetAddressString(),
                         Sector = org.SectorType,
                         ReportingDeadline = org.SectorType.GetAccountingStartDate().AddYears(1).AddDays(-1),
-                        ScopeStatus = org.GetCurrentScope()?.ScopeStatus,
-                        HasSubmitted = org.LatestReturn?.Status == ReturnStatuses.Submitted
```

Comment:
I've realised that these two columns don't make sense out of context.

This is the "Orphan Organisations" download.
This is just a list of the organisations without any users.
It is not being created in the context a particular reporting year.

For each orphan organisation, we give the ScopeStatus (is it in scope) and HasSubmitted (have they submitted).
But, both of these properties only make sense within the context of a particular reporting year.
So, if we give these columns, we're implicitly giving them for a particular reporting year.
This implicit year is different between ScopeStatus and HasSubmitted and can be different between each row - so is useless.

I've removed them for now (they can be joined in Excel if needed).
If we need them in future, we can get more details about the user requirement and build something that works well for them.
But, for the moment, this change is just to stop the data being wrong.

Reply:
I agree and think you have taken the correct approach here